solution3.txt

```diff
@@ -0,0 +1,12 @@
+with open("BCA.txt","w") as file_write:
+    file_write.writelines(data)
+
+line_count = 0
+with open("BCA.txt","r") as file_read,\

```

Comment:
comment this line and rewrite it another way.

Reply:
open to file and read.
f=open("bca.txt",'r',newline='')